test/unit/chips/ChipSet.test.tsx

```diff
@@ -120,30 +120,30 @@ test(
         <div id='1' />
       </ChipSet>
     );
-    wrapper.state().foundation.select = td.func();
+    wrapper.state().foundation!.select = td.func<(chipId: string) => null>();
     const selectedChipIds = ['1'];
     wrapper.setState({selectedChipIds});
     wrapper.instance().initChipSelection();
-    td.verify(wrapper.state().foundation.select('1'), {times: 1});
+    td.verify(wrapper.state().foundation!.select('1'), {times: 1});
   }
 );
 
 test('#handleSelect calls foundation.handleChipSelection with selectedChipId and selected=true', () => {
   const handleChipSelection = td.func();
-  const foundation = {handleChipSelection};
   const wrapper = shallow<ChipSet>(
     <ChipSet>
       <div id='1' />
     </ChipSet>
   );
+  const foundation = coerceForTesting<any>({handleChipSelection});
```

Comment:
Does `Partial<MDCChipFoundation>` work here?

Reply:
The same error:
```
TS2322: Type 'import("/home/kostenko/w/material-components-web-react/node_modules/@material/chips/chip-set/foundation").MDCChipSetFoundation' is not assignable to type 'import("/home/kostenko/w/material-components-web-react/packages/chips/node_modules/@material/chips/chip-set/foundation").MDCChipSetFoundation'.
```

It can be fixed globally by adding option `--hoist` to `lerna bootstrap`, but I don't know if it is correct to do this in this pull request

Reply:
I'm not sure if we would even want to do that. It sounds like we would only have one version of each component, which could hurt us. This is especially apparent in a case like this TS upgrade. We're about half way with the update, but we release on Tuesday. Its fine to release this way, but hoist might hurt us. 

I'm not 100% about hoist, since I just read about it on lerna docs. Since these are just tests and we're trying to mock the foundation, lets just keep it. Thanks!